client/src/styles/app.css

```diff
@@ -1,3 +1,8 @@
 .icon {
   display: block;
 }
+
+/* this is here to make the buttons inside links not trigger events that may make the click handler for any route binding not work */
+a .button, a button {
+  pointer-events: none;
```

Comment:
Because of this change, the hover style on the back button does not trigger:

<img width="97" alt="bildschirmfoto 2018-07-03 um 13 03 47" src="https://user-images.githubusercontent.com/1810384/42216300-ac5f1a3c-7ec1-11e8-9154-ef58e592311d.png">

Reply:
fixed in: https://github.com/nzzdev/Q-editor/pull/143/commits/5d21157cb00f3d6033adb31dfb67ff1bc886f154

the change was needed because events didn't propertly bubble from the button in case it had no click binding.

this is now fixed so the styles are removed.